### DISCUSSION TITLE
[Feature] Blacklist face

### DISCUSSION THREAD

**Discussion by Nonobis** [immich-app/immich#3259] [Feature Request]
### The feature

Hello,

I thnik an option in web/mobile ui for blacklisting face could be great.

I have many pictures of my child in public park, and face recognition modules pick face from stranger ... I would like to blacklist or remove those face from ui.

Possible ?

Thanks !

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2472
